setup.cfg

```diff
@@ -0,0 +1,2 @@
+[metadata]
+description-file = README.md
```

Comment:
@bitsofbits The readme needs to be translated to RST.  PyPi does not support Markdown.